test/phpunit/pro/test-get-options.php

```diff
@@ -37,6 +45,135 @@ public function test_vk_blocks_get_options() {
 					'new_faq_accordion' => 'disable',
 				),
 			),
+			// デフォルトの表示非表示調整 v0.44.13
+			// https://github.com/vektor-inc/vk-blocks-pro/commit/b17c2ae4097a5530eb88a7fdcf03885d89ece643#diff-9b75d117946ae115e35175fb160346351a75f98e850679a65de42680bd4611e1
+			array(
+				'option_check_target' => 'display_vk_block_template',
+				'option'  => array(
+					'hide_wp_block_template' => true,
+					'hide_vk_block_template' => false,
+				),
+				'correct' => 'display',
+			),
+			// hide_wp_block_template,hide_vk_block_templateは廃止されたので以降のテストでは追加していない
+
+			// New FAQ アコーディオン機能 v0.46.0
+			// https://github.com/vektor-inc/vk-blocks-pro/pull/21
+			array(
+				'option_check_target' => 'new_faq_accordion',
+				'option'  => array(
+					'display_vk_block_template' => 'hide',
+				),
+				'correct' => 'disable',
+			),
+			// 吹き出し線の太さ v0.55.0
+			// https://github.com/vektor-inc/vk-blocks-pro/commit/e0ede110ada73f0eb65af611382d899469b8d84b
+			array(
+				'option_check_target' => 'balloon_border_width',
+				'option'  => array(
+					'display_vk_block_template' => 'hide',
+					'new_faq_accordion' => 'open',
+				),
```

Comment:
```suggestion
				'option'  => array(),
```
同じくここも balloon_border_width の初期値テストなので、balloon_border_width 以外の値は不要かなと。
※ もしかして僕が過去に書いたテストの引用とかだったらごめんなさい。